warden/x/warden/keeper/keychain.go

```diff
@@ -12,12 +13,9 @@ func (k Keeper) deductKeychainFees(
 	keychainAccAddress sdk.AccAddress,
 	keychainFees sdk.Coins,
 	creator sdk.AccAddress) error {
-	if maxKeychainFee.Empty() || keychainFees.Empty() {
-		return fmt.Errorf("fees cannot be empty")
-	}
-
-	if !keychainFees.DenomsSubsetOf(maxKeychainFee) {
```

Comment:
Shouldn't we keep this validation if both maxKeychainFee and keychainFees are not empty?

Reply:
IsAllLTE() should cover all cases